src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -349,13 +259,15 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.CloudConfig == "" {
-		cfg.CloudConfig = chainedCloudConfigUrl
+		cfg.CloudConfig = chainedCloudConfigURL
 	}
 
-	if cfg.Client != nil {
-		cfg.applyClientDefaults()
+	if cfg.Client == nil {
+		cfg.Client = &client.ClientConfig{}
```

Comment:
This one is interesting. It seems like whenever we want to apply the defaults we also want client defaults. Unless we add Client to config, though, we won't get those. I didn't have any immediate reason for this other than it repeatedly tripping up tests, but it just seems like what we want here.